### PR TITLE
Modify parameters for GitHub App

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Adjust parameters input for GitHub App workflows.

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -2202,9 +2203,12 @@ components:
         github_installation_id:
           type: integer
           description: GitHub App Installation ID.
-        origin:
+        github_base_repo_url:
           type: string
           description: URL of the GitHub repository containing the Pull Request.
+        github_head_url:
+          type: string
+          description: URL of the Pull Request's head commit.
```
Comment (goern) [inline openapi/openapi.yaml]: its a head_repo_url

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -2202,9 +2203,12 @@ components:
         github_installation_id:
           type: integer
           description: GitHub App Installation ID.
-        origin:
```
Comment (fridex) [inline openapi/openapi.yaml]: Can we keep origin as discussed?

Comment (pacospace) [reply openapi/openapi.yaml]: We can but we won't use it :)

Comment (fridex) [reply openapi/openapi.yaml]: We will (we already use it) - in syncs to keep track of "origin" of software stacks in a queryable form.

Comment (pacospace) [reply openapi/openapi.yaml]: right, let me change it :)

**Pull request merged**